project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -153,6 +142,26 @@ function c4m_user_notifications_moderated_group_notify_user_membership_rejected(
 }
 
 /**
+ * Email notification to group owner, when group is approved.
+ */
+function c4m_user_notifications_notify_user_group_approved(&$form, &$form_state) {
+  $gid = $form_state['values']['gid'];
+  $admin_comment = $form_state['values']['message_from_admin'];
+  $group = node_load($gid);
+
+  $message = message_create('c4m_group_request_approved', array('uid' => $group->uid));
```

Comment:
create the message on an external function and use it also on the `hook_node_update()`